### ISSUE TITLE
Full Custom Fights

### ISSUE THREAD

**Issue by BloomChara** [Jcw87/c2-sans-fight#29]
I was messing with custom attacks and found i can make the menu bones pop up. I thought to myself "hey, what if i can make a full custom battle?" I then went and tried to get the menu to pop up, which i was unable to do. I was wondering if there is, or will be, any way to make full battles? possibly with custom healing items.

**Comment by UnderrepeatSans** [comment]
Same idea! We totally need this! Also, to add on, what about custom sprites? Or custom HP and LV?    If not, it's cool.

**Comment by Jcw87** [comment]
The engine I made this in (Construct 2) wasn't made to be customized after export, so adding mod support wouldn't be easy. The only built-in support for loading external art can only replace existing animation frames on existing objects. You also can't add new object types. Any attempt to add support for this kind of stuff would involve digging deep into the engine, messing with things that you aren't intended to mess with, and can introduce compatibility problems when changing engine versions.

The custom attacks are possible because they don't require adding new art or new objects (and because I went out of my way to allow attacks to be defined from external files).
